Clinical-Genomics/scout
# Extend clinvar form (fix #1340)

Improved the Clinvar submission form.

**Changes from the current master to be tested**
1. When clicking on the 'Submit to clinvar' button on a variant (must be a pinned variant) then the clinvar submission form opens on a new window.
1. Variant data form now contains both **HPO** and **OMIM** terms and the input becomes checkboxes instead of text fields. Example with on the left the old form and on the right the new one:

![image](https://user-images.githubusercontent.com/28093618/67676079-12062200-f981-11e9-9226-38f83af18cd7.png)
1. All options should be available for the zygosity in the casedata form
1. Clinical feature comment and Evidence citation fields are gone from the casedata form
1. "Comment on clinical significance" is now available in the variant data form.
1. Introduced a clinical features field in casedata form that behaves as in point 2 but has only selectable HPO terms

**How to test**:
1. Save a snv and a sv variant as clinvar submission objects, fill in all available fields and check that exported excel documents contain all the provided info.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
